scripts/v1beta1/build.sh

```diff
@@ -112,32 +112,33 @@ echo -e "\nBuilding median stopping rule...\n"
 docker build --platform "linux/$ARCH" -t "${REGISTRY}/earlystopping-medianstop:${TAG}" -f ${CMD_PREFIX}/earlystopping/medianstop/${VERSION}/Dockerfile .
 
 # Training container images
+echo -e "\nBuilding training container images..."
+
 if [ ! "$ARCH" = "amd64" ]; then
-  echo -e "\nTraining container images are supported only amd64."
+  echo -e "\nSome training container images are supported only amd64."
 else
 
-  echo -e "\nBuilding training container images..."
-
   echo -e "\nBuilding mxnet mnist training container example...\n"
   docker build --platform linux/amd64 -t "${REGISTRY}/mxnet-mnist:${TAG}" -f examples/${VERSION}/trial-images/mxnet-mnist/Dockerfile .
 
-  echo -e "\nBuilding Tensorflow with summaries mnist training container example...\n"
-  docker build --platform linux/amd64 -t "${REGISTRY}/tf-mnist-with-summaries:${TAG}" -f examples/${VERSION}/trial-images/tf-mnist-with-summaries/Dockerfile .
-
   echo -e "\nBuilding PyTorch mnist training container example...\n"
   docker build --platform linux/amd64 -t "${REGISTRY}/pytorch-mnist:${TAG}" -f examples/${VERSION}/trial-images/pytorch-mnist/Dockerfile .
 
   echo -e "\nBuilding Keras CIFAR-10 CNN training container example for ENAS with GPU support...\n"
   docker build --platform linux/amd64 -t "${REGISTRY}/enas-cnn-cifar10-gpu:${TAG}" -f examples/${VERSION}/trial-images/enas-cnn-cifar10/Dockerfile.gpu .
 
-  echo -e "\nBuilding Keras CIFAR-10 CNN training container example for ENAS with CPU support...\n"
-  docker build --platform linux/amd64 -t "${REGISTRY}/enas-cnn-cifar10-cpu:${TAG}" -f examples/${VERSION}/trial-images/enas-cnn-cifar10/Dockerfile.cpu .
-
   echo -e "\nBuilding PyTorch CIFAR-10 CNN training container example for DARTS with CPU support...\n"
   docker build --platform linux/amd64 -t "${REGISTRY}/darts-cnn-cifar10-cpu:${TAG}" -f examples/${VERSION}/trial-images/darts-cnn-cifar10/Dockerfile.cpu .
 
   echo -e "\nBuilding PyTorch CIFAR-10 CNN training container example for DARTS with GPU support...\n"
   docker build --platform linux/amd64 -t "${REGISTRY}/darts-cnn-cifar10-gpu:${TAG}" -f examples/${VERSION}/trial-images/darts-cnn-cifar10/Dockerfile.gpu .
+
 fi
 
+echo -e "\nBuilding Tensorflow with summaries mnist training container example...\n"
```

Comment:
Are  arm64 images built only for `tf-mnist-with-summaries` and `enas-cnn-cifar10` ?

Reply:
Yes, all trial images for amd64 except `tf-mnist-with-summaries` and `enas-cnn-cifar10` can work on the M1 Mac since we can emulate x86_64 in [docker desktop with Rosetta2](https://docs.docker.com/desktop/mac/apple-silicon/).

Would you like to make all images conform to arm64 in this PR?

Reply:
I think, we can fix later as well as it works now. We need to cleanup scripts for building images of different architecture.(rather than having arch checks per image etc)

Reply:
It makes sense. @johnugeorge 
It might be better to introduce a multi-arch build after the next Katib release.

ref: https://docs.docker.com/desktop/multi-arch/

I will create an issue to keep tracking this feature.